src/serverdlg.cpp

```diff
@@ -598,6 +598,14 @@ void CServerDlg::OnSysTrayActivated ( QSystemTrayIcon::ActivationReason ActReaso
     {
         ShowWindowInForeground();
     }
+
```

Comment:
I'd prefer
```
#ifdef _WIN32
if ( ActReason == QSystemTrayIcon::Trigger || ActReason == QSystemTrayIcon::DoubleClick )
#else
if ( ActReason == QSystemTrayIcon::DoubleClick )
#endif
```
over the duplication - that's partly why the code format keeps the `if` on its own line, I think.

Reply:
Thank you for the feedback, indeed makes sense to do so.  Will update, test and update the PR.